phy/scripts/phy_script.py

```diff
@@ -45,6 +45,8 @@ def error(self, message):
   phy -v                display the version of phy
   phy describe my_file.kwik
                         display information about a Kwik dataset
+  phy spikesort my_params.prm
+                        run the whole suite (spike detection and clustering)
   phy detect my_params.prm
                         run spike detection on a parameters file
   phy cluster-auto my_file.kwik --num-clusters-start=100
```

Comment:
that option would be good. In any case I suppose most users will just use `phy spikesort` instead